pkgs/tools/text/ugrep/default.nix

```
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, stdenv, pcre2, boost, zlib, lzham }:
```

```suggestion
{ stdenv, fetchFromGitHub, boost, bzip2, lz4, pcre2, xz, zlib}:
```
We can add the other optional deps and I think `lzham` may be the wrong package?